### PR TITLE
Extend iframe with a new allowusermedia attribute (issue: #268)

### PR DESCRIPTION
**Pull request by adam-be**

### CODE REVIEW

**Comment by stefhak** [comment]
@adam-be please ping the list and ask people to review.

File: getusermedia.html [hunk 1]
```diff
@@ -2877,6 +2888,36 @@
       </div>
     </section>
     <section>
+      <h2>User Media in an IFrame</h2>
+      <p>There are some special circumstances when an [[!HTML51]]
+      <code><a href="https://www.w3.org/TR/2015/WD-html51-20150506/semantics.html#the-iframe-element">
+      iframe</a></code> wants to use user media. The iframe needs explicit
```
Comment (martinthomson) [inline getusermedia.html]: s/use/access/ ?  "use user" doesn't flow very nicely

File: getusermedia.html [hunk 2]
```diff
@@ -2877,6 +2890,38 @@
       </div>
     </section>
     <section>
+      <h2>User Media in an IFrame</h2>
+      <div class="note">The topic of this section is under discussion and may
+      be subject to change.</div>
+      <p>There are some special circumstances when a cross-origin [[!HTML51]]
```
Comment (alvestrand) [inline getusermedia.html]: Delete "special". I think it'll be fairly common.

File: getusermedia.html [hunk 3]
```diff
@@ -2877,6 +2890,38 @@
       </div>
     </section>
     <section>
+      <h2>User Media in an IFrame</h2>
+      <div class="note">The topic of this section is under discussion and may
+      be subject to change.</div>
```
Comment (alvestrand) [inline getusermedia.html]: Suggest alternate text: NOTE: A general approach to the subject of passing permissions to iframes is being discussed in other fora. The result of that may give other means of passing permissions to an iframe.

File: getusermedia.html (hunk 2 above)
Comment (stefhak) [inline getusermedia.html]: I'd propose to go one step further. Don't talk about circumstances, just describe that the embedding page must explicitly permit for a cross-origin iframe to be able to request user media.

File: getusermedia.html (hunk 3 above)
Comment (stefhak) [inline getusermedia.html]: Given Martin's comment about not being sure that 'require delegation' will fly we should perhaps stick with the original note.

Comment (adam-be) [inline getusermedia.html]: We can always update the note to reflect the ongoing discussion. I think the current one works for now.

**Comment by adam-be** [comment]
Anyone opposed to merging this one and continue to work on it upstream?

**Comment by alvestrand** [comment]
Let's merge it.

**Comment by stefhak** [comment]
+1, let's merge this.

**Pull request merged**